contractcourt/htlc_outgoing_contest_resolver_test.go

```diff
@@ -165,8 +165,11 @@ func (i *outgoingResolverTestContext) resolve() {
 		}
 	}()
 
-	// Notify initial block height.
-	i.notifyEpoch(testInitialBlockHeight)
+	// Notify initial block height. This is not needed when the output
+	// has been spent before the resolver started.
+	if notifyInitialBlockHeight {
+		i.notifyEpoch(testInitialBlockHeight)
```

Comment:
In production, there will always be an initial block if registering for block epoch with `nil`. So I am not sure if it is a good idea to test different behavior.